DevelopmentProposals/template.md

```diff
@@ -0,0 +1,61 @@
+## Project: TITLE
+***OpenSim Design Document***
+
+### Overview
+
+Just a few sentences that describes the purpose of this project. 
```

Comment:
References to an issue (or issues) are ideal and that information does not have to be repeated here. In the case that this is a new proposed feature (with no related issues) then a solid motivation for the proposal is important.